examples/server_with_celery/example_app/celery.py

```diff
@@ -1,4 +1,8 @@
-from celery import Celery
-from flask_log_request_id.extras.celery import RequestIDAwareTask
+from celery import Celery, signals
+from flask_log_request_id.extras.celery import add_request_id_header, RequestIDAwareTask
 
+# Either sublcass all tasks from RequestIDAwareTask
 celery = Celery(task_cls=RequestIDAwareTask)
+
+# Or alternatively register the provided signal handler
+signals.before_task_publish.connect(add_request_id_header)
```

Comment:
I was wondering if is better to create a function that will hide this techical detail of how to "connect" `flask-log-request-id` and celery. Probably something like `configure_celery(celery_app:Celery)->None` and use that instead.